whisper_timestamped/transcribe.py

```diff
@@ -630,6 +639,13 @@ def cli():
     parser.add_argument("--model_dir", default=None, help="The path to save model files; uses ~/.cache/whisper by default", type=str)
     parser.add_argument("--device", default="cuda:0" if torch.cuda.is_available() else "cpu", help="device to use for PyTorch inference")
     parser.add_argument("--output_dir", "-o", default=None, help="directory to save the outputs", type=str)
+
+    parser.add_argument("--csv", default=True, help="directory to save the outputs", type=str2bool)
+    parser.add_argument("--json", default=False, help="directory to save the outputs", type=str2bool)
+    parser.add_argument("--srt", default=True, help="directory to save the outputs", type=str2bool)
+    parser.add_argument("--txt", default=True, help="directory to save the outputs", type=str2bool)
+    parser.add_argument("--vtt", default=True, help="directory to save the outputs", type=str2bool)
+
```

Comment:
Nice addition :+1: 
I like the fact that it's doing (almost) the same as `whisper` by default. And that we can offer more flexibility.

Just, shouldn't the help messages be "whether to save the outputs in XXX format"?
(everything being saved in the same folder specified by --output_dir)

About the json not being an output by default, a note should be added in the README (which shows this output, that is the more complete and well structured)